base/cmd.jl

```diff
@@ -262,6 +262,15 @@ setenv(cmd::Cmd, env::Pair{<:AbstractString}...; dir=cmd.dir) =
     setenv(cmd, env; dir=dir)
 setenv(cmd::Cmd; dir=cmd.dir) = Cmd(cmd; dir=dir)
 
+# split environment entry string into before and after first `=` (key and value)
+function splitenv(e::String)
+    i = findfirst('=', e)
```

Comment:
```suggestion
    i = findfirst('=', e, 2)
```

Reply:
Oh, does `==x` define an env variable called `=` set to `x`?

Reply:
no, that is illegal, but `=C:=\` defines the current working directory for `C:` to be `\`